### ISSUE TITLE
Litecoin One Project

### ISSUE THREAD

**Issue by Gulzaman786** [MetaMask/eth-phishing-detect#2406]
MetaMask detect That Litecoin One Website could currently compromise your security and has prevented you from interacting with it, But we believes that our website incorrectly flagged And blocked by MetaMask .
Pictures of our web site I believe you is not real but the reason for this is that the cryptocurrency is banned in our country ,
But there is nothing to worry about our team and we working very hard on our project according to our roadmap 
It is not necessary that only pictures must be real but It is important that the people behind the project should be good. Our team is absolutely active and promoting very well, Our whitepaper is a bit weaker but in the future we will try to make it better .
Hope you understand all the situation
So open your heart and give us the courage, we will not allow our plan to fail
We will improve it further, and it will be very useful for our community in the future.
Kindly review again our site www.litecoinone.org and approve as soon as possible hope soon will solve this problem.
Thank you

**Comment by 409H** [comment]
Closing. Duplicate of https://github.com/MetaMask/eth-phishing-detect/issues/2376
